app/routes/events/components/EventAdministrate/Attendees.js

```diff
@@ -101,6 +101,10 @@ export default class Attendees extends Component<Props, State> {
       (reg) => reg.adminRegistrationReason !== '' && reg.pool
     ).length;
 
+    const paidCount = registered.filter(
+      (reg) => reg.paymentStatus == 'succeeded' && reg.pool
```

Comment:
According to our linter, `==` is expected to be `===`. You can verify by running `yarn lint`.

Reply:
Good?